catalog/security/banner-marking/src/main/java/org/codice/alliance/security/banner/marking/BannerMarkings.java

```diff
@@ -27,6 +27,7 @@
 import org.apache.commons.lang.StringUtils;
 
 public class BannerMarkings implements Serializable {
+
```

Comment:
❔ was this just an extra space added?

Reply:
Yeah, though I just realized this wasn't present in the 1.6.x PR so I removed it via a rebase.